src/client/entity-editor/series-section/series-section.tsx

```diff
@@ -0,0 +1,159 @@
+ * specific to the series entity. The intention is that this component is
+ * rendered as a modular section within the entity editor.
+ *
+ * @param {Object} props - The properties passed to the component.
+ * @param {Array} props.seriesOrderingTypes - The list of possible ordering
+ * 		  types for a series.
+ * @param {number} props.orderTypeValue - The ID of the ordering type currently selected for
+ *        the series.
+ * @param {string} props.seriesTypeValue - The value of the entity type currently selected for
+ *        the series.
+ * @param {Immutable.List<any>} props.relationships - The list of relationships conatined by
+ * 		  the series.
+ * @param {Function} props.onOrderTypeChange - A function to be called when
+ *        a different ordering type is selected.
+ * @param {Function} props.onSeriesTypeChange - A function to be called when
+ *        a different series type is selected.
+ * @returns {ReactElement} React element containing the rendered
+ *        SeriesSection.
+ */
+function SeriesSection({
+	seriesOrderingTypes,
+	orderTypeValue,
+	seriesTypeValue,
+	relationships,
+	onOrderTypeChange,
+	onSeriesTypeChange
+}: Props) {
+	const relationshipsObject = relationships.toJS();
+	const relationshipsArray = Object.values(relationshipsObject);
+	const seriesOrderingTypesForDisplay = seriesOrderingTypes.map((type) => ({
+		label: type.label,
+		value: type.id
+	}));
+	const seriesTypesForDisplay = ['Author', 'Work', 'Edition', 'EditionGroup', 'Publisher'].map((entity) => ({
+		label: entity,
+		value: entity
+	}));
+	return (
+		<div>
+			<h2>
+				What else do you know about the Series?
+			</h2>
+			<p className="text-muted">
+				All fields are mandatory — select the option from dropdown
+			</p>
+			<Row>
+				<Col md={6} mdOffset={3}>
+					<CustomInput
+						label="Ordering Type"
+						tooltipText="Ordering Type of the Series Entity"
+					>
+						<Select
+							backspaceRemoves={false}
+							clearable={false}
+							deleteRemoves={false}
+							instanceId="seriesOrderingType"
+							options={seriesOrderingTypesForDisplay}
+							value={orderTypeValue}
+							onChange={onOrderTypeChange}
+						/>
+					</CustomInput>
+					<CustomInput label="Series Type" tooltipText="Entity Type of the Series">
+						<Select
+							backspaceRemoves={false}
+							clearable={false}
+							deleteRemoves={false}
+							disabled={Boolean(relationshipsArray.length)}
+							instanceId="SeriesType"
+							options={seriesTypesForDisplay}
+							value={seriesTypeValue}
+							onChange={onSeriesTypeChange}
+						/>
+					</CustomInput>
+				</Col>
+			</Row>
+		</div>
+	);
+}
+SeriesSection.displayName = 'SeriesSection';
+
+function mapStateToProps(rootState): StateProps {
+	const state = rootState.get('seriesSection');
+
+	return {
+		orderTypeValue: state.get('orderType'),
+		relationships: rootState.getIn(['relationshipSection', 'relationships']),
```

Comment:
You should be able to do something like this if I'm not mistaken:
```suggestion
		relationships: rootState.getIn(['relationshipSection', 'relationships']).toArray(),
```